client/header/activity-panel/panels/orders.js

```diff
@@ -5,33 +5,57 @@
 import { __, _n, sprintf } from '@wordpress/i18n';
 import { Button } from '@wordpress/components';
 import { Fragment } from '@wordpress/element';
+import { compose } from '@wordpress/compose';
+import { withSelect } from '@wordpress/data';
```

Comment:
When you rebase from master, you'll want to use `fresh-data` now that its available (https://github.com/woocommerce/wc-admin/pull/833).
```js
import withSelect from 'wc-api/with-select';
```

Reply:
Done in e6128de.